content/compute/boot-from-volume.md

```diff
@@ -0,0 +1,199 @@
+disk size allocation is no longer relevant since we'll be using a volume as our root disk.
+
+```
+$ nova flavor-list
+
++--------------------------------------+------------+-----------+------+-----------+------+-------+-------------+-----------+
+| ID                                   | Name       | Memory_MB | Disk | Ephemeral | Swap | VCPUs | RXTX_Factor | Is_Public |
++--------------------------------------+------------+-----------+------+-----------+------+-------+-------------+-----------+
+| 1791eb32-68a1-4ec9-ac8d-e2374ca9c909 | 1 GB       | 1024      | 30   | 0         |      | 1     | 1.0         | True      |
+| 34719f08-452d-4e8d-a29f-548d82985477 | 4 GB - HC  | 4096      | 60   | 0         |      | 4     | 1.0         | True      |
+| 569d85f5-647c-420d-98e6-85149e6eebf6 | 32 GB - HM | 32768     | 60   | 0         |      | 6     | 1.0         | True      |
+| 64c94219-1452-4934-a09a-d702c9370c3d | 8 GB - HM  | 8192      | 60   | 0         |      | 2     | 1.0         | True      |
+| 677cac99-6a5a-4c6a-9884-23338d01c04d | 2 GB       | 2048      | 60   | 0         |      | 1     | 1.0         | True      |
+| 75518817-da1b-4fbf-bd11-dffce8e73b49 | 8 GB - HC  | 8192      | 60   | 0         |      | 8     | 1.0         | True      |
+| 885f9496-5de5-4894-bab9-98ef88e4892c | 8 GB       | 8192      | 120  | 0         |      | 4     | 1.0         | True      |
+| b5ae8e6f-03f3-4146-9add-6e84e6944ead | 4 GB       | 4096      | 80   | 0         |      | 2     | 1.0         | True      |
+| baec2dce-b1ea-4898-a259-9a3bf66f6262 | 512 MB     | 512       | 10   | 0         |      | 1     | 1.0         | True      |
+| cdb1ff78-66ea-4848-a93e-f843a75c7f59 | 16 GB - HC | 16384     | 60   | 0         |      | 12    | 1.0         | True      |
+| d0e71e26-1d9e-4476-bf8b-11d32dc30483 | 16 GB - HM | 16384     | 60   | 0         |      | 4     | 1.0         | True      |
+| e989b4c4-4aa2-4b5c-8086-d3f407fb131f | 16 GB      | 16384     | 200  | 0         |      | 6     | 1.0         | True      |
+| f021e071-261c-43e3-b1d7-834bc714e06b | 32 GB      | 32768     | 200  | 0         |      | 8     | 1.0         | True      |
++--------------------------------------+------------+-----------+------+-----------+------+-------+-------------+-----------+
+
+$ neutron net-list
+
++--------------------------------------+--------------+------------------------------------------------------+
+| id                                   | name         | subnets                                              |
++--------------------------------------+--------------+------------------------------------------------------+
+| 7bc79d7b-0d04-488b-9513-b4695988840e | Demo Network | 28359fd5-5858-4ad3-94f4-673a31927603 10.200.100.0/24 |
++--------------------------------------+--------------+------------------------------------------------------+
+
+```
+
+In this example, we'll use the *4 GB* flavour and boot our instance on the *Demo Network*. 
```

Comment:
note that `boot-from-vol` is the name of your instance, and not an additional required argument.